hologram_dashboard/fetcher/hologram_data_source_availability_fetcher.cc

```diff
@@ -34,4 +39,27 @@ void HologramDataSourceAvailabilityFetcher::
                                                 &hologram_configs_));
 }
 
+void HologramDataSourceAvailabilityFetcher::FetchFromDatabase() {
+    // TODO(alexanderlin): Implement.
+}
+
+void HologramDataSourceAvailabilityFetcher::GetStatus(absl::CivilMinute date) {
+    // TODO(alexanderlin): Implement.
+    // for(const std::pair<std::string, std::string>& it : system_to_cell_map_) {
+    //     for(int i = 0; i < hologram_configs_.data_source_config_size(); ++i) {
+    //     UpdateProto(it.first, date, hologram_configs_.data_source_config(i), status);
+    // }
```

Comment:
Why you are commenting the code?

Reply:
this is just pseudo code so i didn't want it to interfere with other code's compilation

Reply:
This is not a very good practice. 

But considering this part of comment is short, it is fine to leave it here this time. 

In follow-up cls, please don't do pseudo code like this.

Reply:
got it